### ISSUE TITLE
Add support for WordPress (CMS - content management system)

### ISSUE THREAD

**Issue by SergeyKleyman** [elastic/apm-agent-php#385]
- [ ] Automatically discover service name
- [ ] Automatically name a transaction based the selected route
- [ ] Capture errors escaping from the application to the framework

**Comment by archon810** [comment]
Yes please. We'll be the first to give it a go once it's stable.

**Comment by ramon-yoast** [comment]
@SergeyKleyman could you elaborate why the milestones have been bumped twice from 8.3 to 8.5?
